src/@types/web-activities/activity-ports.d.ts

```diff
@@ -69,16 +69,22 @@ declare module 'web-activities/activity-ports' {
    * for a successful result, a cancelation or a failure.
    */
   interface ActivityResult {
+    readonly code: ActivityResultCode;
+    readonly data: unknown;
+    readonly mode: ActivityMode;
+    readonly origin: string;
+    readonly originVerified: boolean;
+    readonly secureChannel: boolean;
     readonly ok: boolean;
     readonly error: Error | null;
 
     constructor(
-      readonly code: ActivityResultCode,
```

Comment:
Note: This shorthand doesn't work in `.d.ts` files 😭

Reply:
Should we clean up the other places we're using that syntax, like lines 113-115? (Note, private fields should not need to be declared in the types file, so those only need to specify the constructor interface).